examples/with-emotion/pages/index.js

```diff
@@ -69,15 +69,11 @@ export default () => {
 
   return (
     <div>
-      <Basic>
-        Cool Styles
-      </Basic>
+      <Basic>Cool Styles</Basic>
       <Combined>
         With <code>:hover</code>.
       </Combined>
-      <Animated animation={bounce}>
-        Let's bounce.
-      </Animated>
+      <Animated animation={bounce}>Let's bounce.</Animated>
```

Comment:
Looks like prettier made this change to one line, I can revert this back if you'd like. cc @tkh44